src/ai/susi/mind/SusiSkill.java

```diff
@@ -500,6 +500,7 @@ public static JSONObject getSkillMetadata(String model, String group, String lan
         skillMetadata.put("dynamic_content", false);
         skillMetadata.put("examples", JSONObject.NULL);
         skillMetadata.put("skill_rating", JSONObject.NULL);
+        skillMetadata.put("skill_identifier", JSONObject.NULL);
```

Comment:
@Orbiter There is already a key named `skill_name` in the metedata object, that shows the display name of the skill. By this PR, @arundhati24 I think is trying to send the filename(the last children of the path, i.e,  `/<path_to_skill>/skillname.txt`) which the user saves while creating the skill. Should the filename be sent as `skill_path` or `skill_file` in the response? What do you suggest?

Reply:
@Orbiter @akshatnitd Yes, what I need is the last child of the path. For e.g. : "capital" in the following code block. The skill_name is "Capital" while what I require is the key associated with the skill metadata object i.e. "capital". This is needed for other tasks like sending the five star rating via fiveStarRateSkill.json API where we need to send this key as a query.

```
"capital": {
      "model": "general",
      "group": "Knowledge",
      "language": "en",
      "developer_privacy_policy": null,
      "descriptions": "A skill to tell user about capital of any country.",
      "image": "images/capital.png",
      "author": "chashmeet singh",
      "author_url": "https://github.com/chashmeetsingh",
      "author_email": null,
      "skill_name": "Capital",
      "terms_of_use": null,
      "dynamic_content": true,
      "examples": ["What is the capital of India?"],
      "skill_rating": {
        "negative": "0",
        "positive": "0",
        "stars": {
          "one_star": 0,
          "four_star": 0,
          "five_star": 1,
          "total_star": 1,
          "three_star": 0,
          "avg_star": 5,
          "two_star": 0
        },
        "feedback_count": 1
      },
      "creationTime": "2018-03-17T17:11:59Z",
      "lastAccessTime": "2018-06-18T17:34:52Z",
      "lastModifiedTime": "2018-03-17T17:11:59Z"
    }
```

Reply:
If you identify the skill by the skill path, call it skill_path. If you identify the skill by the name only, it could be skill_name but we already had a mess in the past because the name of the skill and the file name had been messed up. Please choose any name that is meaningful and use a way to address the skill in such a way that there cannot be conflicts because skills in different paths may get the same name.